src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -0,0 +1,412 @@
+# -*- coding: utf-8 -*-
+"""
+This file contains the O(as1aem1) Altarelli-Parisi splitting kernels.
+
+These expression have been obtained using the procedure described in the
```

Comment:
Too much copy&paste ;-)

Reply:
Yeah, definitely that's the procedure for `as1` xD